webpack/assets/javascripts/react_app/components/common/DateTimePicker/DateComponents/helpers.js

```diff
@@ -30,13 +30,16 @@ export const getMonthStart = date => {
 
 export const getWeekStart = date => addDays(date, (7 - date.getDay()) % 7);
 
+export const parseLocale = locale => locale.replace('_', '-');
```

Comment:
I don't think the component should be responsible for parsing the locale prop, but the consumer needs to verify that.
As you can see [here](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/I18n.js#L43) the locale is modified by `replace' due to Jed requirements. making a replace over a replace looks odd to me.

Reply:
We should choose one format and be consistent, what is rendering the locale? It seems more common to be a dashed styled in javascript. Can't we just transform the locale key before rendering the `localedata`? That way we wouldn't need the replace neighter here, nor in the documentLocale.

Reply:
You're right, added a function in the helper file but without the replace